src/workerd/api/node/BUILD.bazel

```diff
@@ -26,10 +26,124 @@ kj_test(
     deps = ["//src/workerd/tests:test-fixture"],
 )
 
-[wd_test(
-    src = f,
+wd_test(
```

Comment:
some tests are large, so rather than deal with excludes I expanded this. Hope you don't mind @anonrig